BitwardenShared/Core/Auth/Models/Request/ResendNewDeviceOtpRequestModel.swift

```diff
@@ -0,0 +1,17 @@
+import Foundation
+import Networking
+
+// MARK: - ResendNewDeviceOtpRequestModel
+
+/// API request model for re-sending the device verification code to email.
+///
+struct ResendNewDeviceOtpRequestModel: JSONRequestBody {
+    // MARK: Static Properties
+
+    static let encoder = JSONEncoder()
```

Comment:
🤔 Do you need this default encoder? Because by default `JSONRequestBody` [has](https://github.com/bitwarden/ios/blob/main/BitwardenShared/Core/Platform/Services/API/Extensions/JSONRequestBody%2BBitwarden.swift#L6):
`static var encoder: JSONEncoder { .defaultEncoder }`

Reply:
Indeed, they need to be updated because the one in the extension was introduced at a later moment.